ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -19,9 +19,13 @@
 import UIKit
 import ownCloudSDK
 
-class ClientQueryViewController: UITableViewController, Themeable {
-	var core : OCCore?
-	var query : OCQuery?
+class ClientQueryViewController: UIViewController {
```

Comment:
Why did you change the superclass from `UITableViewController` to `UIViewController` and set up the tableView yourself?

Reply:
The problem this PR solves is that when you don't have any files, the sort bar was hidden by the _No Files_ view. 

There were two approaches:

-  Make the _No Files_ view top constraint constant to be 40 pt (the height of the sort bar) to not hide it with this view. Then the scrolling effect to reload scrolls the sort bar under the _No files_ view and the effect was not beauty.

- Make the `ClientQueryViewController` be a child of `UIViewController` and make the sort bar part of the view controller view.
 The only downside of this is that you need to create the `tableView`  yourself and implement the `UITableViewDelegate` and `UITableViewDataSource`. This way you can have the sort bar independent of the `tableView` and present the _No Files_ view above only the `tableView` allowing the user to push the _create folder_ button and not making any rare scroll with the sort bar.